src/api/proxy/CacheProxy.ts

```diff
@@ -0,0 +1,70 @@
+import NodeCache from 'node-cache'
+
+export class CacheProxy<T> {
+  protected api: T
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
is this still necessary?

Reply:
Nope, forgot to remove. There are probably more like that, will remove them.